package.json

```diff
@@ -23,10 +23,10 @@
     "build": "yarn build:pre-tsc && yarn build:tsc && yarn build:post-tsc",
     "build:clean": "yarn clean && yarn build",
     "build:tsc": "tsc --build",
-    "build:pre-tsc": "yarn workspaces foreach --parallel --topological run build:pre-tsc",
-    "build:post-tsc": "yarn workspaces foreach --parallel --topological run build:post-tsc",
-    "clean": "yarn workspaces foreach --parallel run clean",
-    "test": "yarn workspaces foreach --parallel run test",
+    "build:pre-tsc": "yarn workspaces foreach --parallel --topological -v run build:pre-tsc",
```

Comment:
What's the reasoning for using the verbose flag?

Reply:
It displays workspace name along log output, otherwise, especially with parallel commands its hard to figure out from which workspace error came

Reply:
Yeah, I like that 👍 We may want to add that elsewhere too, but out of scope for this.